scripts/pre-commit.sh

```diff
@@ -149,6 +149,10 @@ echo "Deploying cluster-scoped kubefed"
 echo "Running e2e tests against cluster-scoped kubefed"
 run-e2e-tests
 
+echo "Validating KubeFed walkthrough"
+./scripts/deploy-federated-nginx.sh
+kubectl delete ns test-namespace
```

Comment:
Deletion of the test namespace should be done as part of the script, please fix in a follow-up.